public/Projects/IQtheEngineerBigfootGame/info.txt

```diff
@@ -0,0 +1,7 @@
+{
+    name: "Iman Crooks",
+    ghurl: "https://github.com/iqtheengineer",
+    imgName: "assets/bigfoot.gif",

```

Comment:
put the gif inside `public/assets` folder. (All the gifs of different projects are present there)

the imgName should be the gif name only and not the path to the gif. 

For example,

```
imgName: "bigfoot.gif"
```

Reply:
Added commit (a2490ca8f011f008a4acf76996e33220ad5fc7d8) to add game instructions via a prompt message and relocated gif to public/assets folder and renamed to bigfoot.gif. Please let me know if anything else is needed. @SandeepKrSuman

Reply:
Just modify info.txt file

remove ----> `imgName: "assets/bigfoot.gif"`

add ----->  `imgName: "bigfoot.gif"`

Just do this. Everything else is fine.